velox/functions/prestosql/Reduce.cpp

```diff
@@ -104,6 +104,8 @@ class ReduceFunction : public exec::VectorFunction {
     });
 
     // Fix finalSelection at "rows" unless already fixed.
+    const SelectivityVector& finalSelectionRows =
```

Comment:
This code can be simplified if it is moved after the finalSelection and isFinalSelection.

`const auto& finalSelectionRows = *context->finalSelection();`